include/dem/ray_tracing_solver_parameters.h

```diff
@@ -0,0 +1,68 @@
+// SPDX-FileCopyrightText: Copyright (c) 2020-2025 The Lethe Authors
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception OR LGPL-2.1-or-later
+
+#ifndef lethe_ray_tracing_solver_parameters_h
+#define lethe_ray_tracing_solver_parameters_h
+
+#include <core/parameters.h>
+#include <core/parameters_lagrangian.h>
+#include <core/simulation_control.h>
+// #include <core/solid_objects_parameters.h>
+
+/**
+ * @brief Handles all the parameters declared in the parameter handler file.
+ */
+template <int dim>
+class RayTracingSolverParameters
+{
+public:
+  Parameters::Mesh                                mesh;
+  Parameters::Testing                             test;
+  Parameters::Timer                               timer;
+  Parameters::SimulationControl                   simulation_control;
+  Parameters::Lagrangian::InsertionInfo<dim>      particle_insertion_info;
+  Parameters::Lagrangian::ParticleRayTracing<dim> ray_tracing_info;
+  Parameters::Lagrangian::ModelParameters<dim>    model_parameters;
+
+  // Maybe usefull later
```

Comment:
I would suggest removing them. Or at least write useful without a spelling mistake. But I would tend to avoid putting feature comments which you most likely won't have the time to implement